components/ShareLink.tsx

```diff
@@ -0,0 +1,55 @@
+import { useWeb3React } from "@web3-react/core";
+import { useRouter } from "next/router";
+
+export const ShareLink = ({ hasWonGame, shareText, trophyId }) => {
+  const { account } = useWeb3React();
+  const { query } = useRouter();
+  return (
+    <a
+      className="share-button"
+      onClick={() => {
+        const gameTiles = document.getElementsByClassName("game-tile");
```

Comment:
There is a much better way to do this. Because of the virtual dom it's not recommended to select elements like this. WIll fix.